data/presets/highway/primary_link-US-CA.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Primary Link",
+    "name": "{primary_link}",
```

Comment:
Shouldn't it be `"{highway/primary_link}"`?

Reply:
of course. thanks for double-checking!